scripts/kani-run-on-repos.sh

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+# Copyright Kani Contributors
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+
+
+DOCUMENTATION=\
+'kani-run-on-repos.sh -- script to clone and compile multiple remote git repositories with Kani.
+
+USAGE:
+./scripts/kani-top-100.sh path/to/url-list
```

Comment:
Script name needs to be updated.

Reply:
Fixed with 6135a3d8543a50fd54650d972d2450ee23aad63a